tests/plugins/dialog/positioning.js

```diff
@@ -13,8 +13,11 @@
 		},
 		tearDown: function() {
 			var dialog = CKEDITOR.dialog.getCurrent();
-			if ( dialog ) {
+
+			while ( dialog ) {
```

Comment:
Please take a look at comment to `tearDown` in `tests/plugins/dialog/plugins.js`